### PR TITLE
Feature/implement bubblesort

### PR DESCRIPTION
**Pull request by yxyx-github**

### CODE REVIEW

File: src/algorithms/sort/bubbleSort.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    for (let lastElement = numbers.length - 1; lastElement > 0; lastElement--) {
+        for (let pointer = 0; pointer < lastElement; pointer++) {
+            if (numbers[pointer] > numbers[pointer + 1]) {
+                item = numbers[pointer]
+                numbers[pointer] = numbers[pointer + 1]
+                numbers[pointer + 1] = item
+            }
+            pB.step({
+                sortedValues: numbers,
+                highlightedIndices: [pointer, pointer + 1, lastElement],
```
Comment (termitehuegel) [inline src/algorithms/sort/bubbleSort.ts:16]: Please check that pointer +1 != lastElement. Otherwise, this might lead to errors later on.

Comment (termitehuegel) [inline src/algorithms/sort/bubbleSort.ts:16]: I wouldn't highlight lastElement at all. But we can leave it if you want.

**Review by termitehuegel — CHANGES_REQUESTED** [review]

File: src/algorithms/sort/bubbleSort.ts (hunk 1 above)
Comment (yxyx-github) [reply src/algorithms/sort/bubbleSort.ts]: Fixed that

Comment (yxyx-github) [reply src/algorithms/sort/bubbleSort.ts]: Fixed, now there arn't any duplicate highlightings anymore.

**Review by termitehuegel — APPROVED** [review]
looks good

**Pull request merged**